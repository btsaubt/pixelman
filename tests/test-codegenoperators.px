def int main() {
  int a;
  int b; 
  float c;
  float d;
  float e; 

  a = 10;
  b = 2; 
  c = 2.5;
  d = 1.2;
  e = 0.5; 
  
  print(a >> b); 
  print(a << b); 
  
  printb(c+d == 3.7); 
  printb(c-d == 1.3); 
  printb(c*d == 3.0);
  printb(c <= 2.5);
  printb(c <= 3.0);
  printb(c >= 2.0);
  printb(c >= 2.5); 
  printb(c < 3.7);
  printb(3.7 > d); 
  printb(d/e == 2.4); 
  printb(12%5 == 2); 

  return 1; 
}
