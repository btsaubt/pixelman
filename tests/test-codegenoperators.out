2
40
True
True
True
True
True
True
True
True
True
True
True
